configs/application-alpha.conf

```diff
@@ -5,4 +5,4 @@ common-instance = alpha
 com.socrata.coordinator.secondary-watcher = {
   database.database = datacoordinator_alpha
   collocation.group = [alpha, bravo, charlie]
-}
```

Comment:
These are missing newlines my editor just found. May as well correct them now.

Reply:
Yup! Confusing because GitHub shows the curly in red